bin/bench-tx/src/main.rs

```diff
@@ -59,6 +60,7 @@ fn main() -> Result<(), String> {
 // ================================================================================================
 
 /// Runs the default transaction with empty transaction script and two default notes.
+#[allow(clippy::arc_with_non_send_sync)]
```

Comment:
We need this attribute because `TransactionContext` is not `send` and `sync`? If so, could we make it `send` and `sync`?